.prettierignore

```diff
@@ -0,0 +1,6 @@
+dist
+package.json
+package-lock.json
+pnpm-lock.yaml
+.rush/temp/**
```

Comment:
Not that it hurts, but... no Rush, no pnpm. :slightly_smiling_face:

Reply:
oh, yeah, I was trying to have a common ignore file I could copy-pasta around without "worrying about it" :)